scalatest/src/main/scala/org/scalatest/tools/Framework.scala

```diff
@@ -430,16 +431,17 @@ class Framework extends SbtFramework {
           annotationClass.getName
         else
           value
-      }
+      }).asInstanceOf[Array[String | Null]]
```

Comment:
**Overridden java array field**
A Scala field overrides some Java array field. We need to use `Array[T | Null]` instead of `Array[T]` for the overriding to work.